kinesis/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/sinks/KinesisSink.scala

```diff
@@ -228,8 +230,11 @@ class KinesisSink private (
     ()
   }
 
+  type BatchEntry = (ByteBuffer, String)
```

Comment:
Why not a class for easier field access?

Reply:
I probably didn't want to change the original code from EventStorage too much, but I think this is good idea to use case class here! Thanks!